### PR TITLE
[kill] Lab1 lab1 labadin

### PR DESCRIPTION
**Pull request by ntdesmond**
- Time app on Python + FastAPI
- Time app on Node,js + TypeScript + Express

:lets:

### CODE REVIEW

**Review by DOCtorActoAntohich — APPROVED** [review]
I know that you know that you shouldn't push `.env` file, but I'll say it anyway because we're forced to say something. :trollface: 

Other than that, all good 👍

**Comment by ntdesmond** [comment]
i mean, wrong branch lol

File: app_python/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Use the following commmand to start the app:
+
+```sh
+gunicorn --config .\gunicorn.conf.py app:app
+```
+
+### Uvicorn
+
+Gunicorn runs only on UNIX systems. For Windows, you may use Uvicorn directly:
```
Comment (smthngslv) [inline app_python/README.md:37]: for windows you can install linux or buy macbook instead

File: app_python/app.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+
+from fastapi import FastAPI, Depends
+from fastapi.responses import PlainTextResponse
+
+from dependencies import get_real_time
```
Comment (smthngslv) [inline app_python/app.py:6]: use some package name like `myapp.dependencies`

File: app_python/app.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+app = FastAPI()
+
+
+@app.get("/", response_class=PlainTextResponse)
+async def show_time(time: datetime = Depends(get_real_time)):
```
Comment (smthngslv) [inline app_python/app.py:12]: where's type annotation???

**Review by smthngslv — COMMENTED** [review]
Kinda Good

File: app_python/app.py (hunk 3 above)
Comment (ntdesmond) [reply app_python/app.py:12]: > where's type annotation???

`-> str`? ok, got you. Will fix it in the lab 3 if I don't forget

**Comment by justmark0** [comment]
Look fine to me! 😎

**Pull request closed without merging**